presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestJdbcRecordSetProvider.java

```diff
@@ -47,6 +47,7 @@
 @Test
 public class TestJdbcRecordSetProvider
 {
+    private static final JdbcIdentity IDENTITY = new JdbcIdentity("user", ImmutableMap.of());
```

Comment:
nit: empty line after class-level consts

```suggestion
    private static final JdbcIdentity IDENTITY = new JdbcIdentity("user", ImmutableMap.of());

```